### PR TITLE
Enable SSR for some 4.x build configurations

### PR DESCRIPTION
**Pull request by fbiville**
Signed-off-by: Florent Biville <florent.biville@neo4j.com>

### CODE REVIEW

**Review by dotStart — APPROVED** [review]

File: neo4j.py [hunk 1]
```diff
@@ -99,20 +101,22 @@ class Core:
...
         self._index = index
         self._artifacts_path = join(artifacts_path, self.name)
+        self._use_ssr = use_ssr
         self._container = None
 
     def start(self, image, initial_members, network):
         env_map = {
             "NEO4J_dbms_connector_bolt_advertised__address":
-                "%s:%d" % (self.name, 7687),
+                ("%s:%d" % (self.name, 7687)),
```
Comment (robsdedude) [inline neo4j.py]:
```suggestion
                "%s:%d" % (self.name, 7687),
```

**Comment by robsdedude** [comment]
I assume this is still relevant. What's missing (apart from green pipelines :D)?

**Comment by fbiville** [comment]
yep, @bigmontz and I wanted to catch errors early on with SSR in 4.x, since SSR was to be enabled by default in 5.0 (and is).

**Comment by robsdedude** [comment]
Too stale. Feel free to re-open if still relevant.

**Pull request closed without merging**